ui/src/api/jq/mutate/executeQuery.ts

```diff
@@ -1,13 +1,17 @@
-import { JqQueryResult, JqQueryResultType } from "../schema";
+import {
+  ExecuteJqueryPayloadType,
+  JqQueryResult,
+  JqQueryResultType,
+} from "../schema";
 
 import { apiFactory } from "~/api/apiFactory";
 import { encode } from "js-base64";
 import { getMessageFromApiError } from "~/api/errorHandling";
 import { useApiKey } from "~/util/store/apiKey";
 import useMutationWithPermissions from "~/api/useMutationWithPermissions";
 
-export const executeJquery = apiFactory<{ query: string; data: string }>({
-  url: ({ baseUrl }: { baseUrl?: string }) => `${baseUrl ?? ""}/api/jq`,
+export const executeJquery = apiFactory<ExecuteJqueryPayloadType>({
+  url: ({ baseUrl }: { baseUrl?: string }) => `${baseUrl ?? ""}/api/v2/jx`,
```

Comment:
I am a bit confused about the terminology here.

1. Why has "jq" changed to "jx" in the endpoint url (as well as in the payload schema)? Is jx a different format? I guess I am missing some background info here.

2. I think we should avoid Jquery in the name of our types? It looks like the old JavaScript JQuery, which is a different thing. I will make another comment on this in the schema definition file.

There is also the existing hook `useExecuteJQuery` which should maybe be updated to `useExecuteJqQuery`, and the name of this file...